### ISSUE TITLE
Running the file locally has outdated instructions

### ISSUE THREAD

**Issue by spencerrecneps** [azavea/pfb-network-connectivity#473]
Was just trying to run a new analysis by following the instructions in README.RUNNING.md but it appears the instructions are out of date. There is no pfb-analysis directory so these instructions don't work:
```
pushd pfb-analysis
docker build -t pfb-analysis .
popd
```

**Comment by CloudNiner** [comment]
We'll address this eventually, but the replacement command for the three lines you noted above is:
```
docker-compose build analysis
```

**Comment by CloudNiner** [comment]
Addressed by #581
